### ISSUE TITLE
Without jQuery

### ISSUE THREAD

**Issue by jimmywarting** [mattboldt/typed.js#203]
I didn't want to include jQuery just for this plugin, so I created my own version of this.
It's quite scaled down, Typed.js had to many options IMO - only did what was necessary for me

Right now it only works in chrome, opera & FF 52
But with babel it will work like a charm...

For anyone interested here is a [demo](https://jsfiddle.net/2jgvftf2/3)

**Comment by Orbyt** [comment]
I just visited this repository looking to see if it was usable with React. Using jQuery with React is a no-go, which renders this project unusable for those using React.

Perhaps an official port without jQuery for use with React would be a useful addition here, as the current options for typing animations usable with React are not as popular as Typed.js.

**Comment by jimmywarting** [comment]
I tried React before... Never liked it. They change the html syntax to camelCase, even changed some attributes names that didn't even make since to me like using `className` instead of just `class`. And coding html in jsx was something else... 

When you have worked with web and Node for such a long time and you know that all frameworks comes and go but one thing will always stay. Html, css & javascript
So today i mostly only do vanilla coding

But if you want a react port of this then it's fine by me. But you have to do it without my contribution

**Comment by jimmywarting** [comment]
Now when i think of it i would maybe even have extracted out the html part out of my version so it can be used for inputs, objects Data-binding and other DOM elements

```javascript
typed({
	strings: ['Hangout', 'Facebook', 'Web App', 'your email', 'Messenger', 'anything!!!']
	set value(val) {
	  document.querySelector('.element').innerText = val
	}
})
```

**Comment by luisvinicius167** [comment]
I created a library for me, with no jQuery dependency, if someone can take a look: https://ityped.surge.sh
Thanks!

**Comment by sequentialbro** [comment]
@luisvinicius167 

Very sexy, and it works in my React Project!

**Comment by luisvinicius167** [comment]
@sequentialbro Yeah, nice! ;)

**Comment by mattboldt** [comment]
A jQuery-free typed.js has been released under v1.1.6 thanks to this guy: @nikrolls
